aptos-move/framework/aptos-framework/sources/fungible_asset.move

```diff
@@ -82,6 +83,8 @@ module aptos_framework::fungible_asset {
     const EFUNGIBLE_METADATA_EXISTENCE: u64 = 30;
     /// Cannot register dispatch hook for APT.
     const EAPT_NOT_DISPATCHABLE: u64 = 31;
+    /// Flag for Concurrent Supply not enabled
```

Comment:
nit: . is missing and it should be balance, not supply?

Reply:
```suggestion
Flag for Concurrent balance not enabled.
```